_studio/mfx_lib/decode/CMakeLists.txt

```diff
@@ -20,118 +20,161 @@
+    )
+
+list(APPEND sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/mpeg2/src/mfx_mpeg2_decode.cpp
+    ${UMC_CODECS}/mpeg2_dec/src/umc_mpeg2_bitstream.cpp
+    ${UMC_CODECS}/mpeg2_dec/src/umc_mpeg2_decoder.cpp
+    ${UMC_CODECS}/mpeg2_dec/src/umc_mpeg2_decoder_va.cpp
+    ${UMC_CODECS}/mpeg2_dec/src/umc_mpeg2_frame.cpp
+    ${UMC_CODECS}/mpeg2_dec/src/umc_mpeg2_slice.cpp
+    ${UMC_CODECS}/mpeg2_dec/src/umc_mpeg2_splitter.cpp
+    ${UMC_CODECS}/mpeg2_dec/src/umc_mpeg2_utils.cpp
+    ${UMC_CODECS}/mpeg2_dec/src/umc_mpeg2_va_packer.cpp
+    )
+
+list(APPEND sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/vc1/src/mfx_vc1_decode.cpp
+    ${UMC_CODECS}/vc1_common/src/umc_vc1_common.cpp
+    ${UMC_CODECS}/vc1_common/src/umc_vc1_common_tables.cpp
+    ${UMC_CODECS}/vc1_common/src/umc_vc1_common_tables_adv.cpp
+    ${UMC_CODECS}/vc1_common/src/umc_vc1_spl_frame_constr.cpp
+    ${UMC_CODECS}/vc1_common/src/umc_vc1_spl_tbl.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_huffman.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_bitplane.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_bpic.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_bpic_adv.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_frame_descr.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_frame_va.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_ipic.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_ipic_adv.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_pic_com.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_pic_com_adv.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_ppic.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_ppic_adv.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_seq.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_skipping.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_task_store.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_dec_vopdq.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_video_decoder.cpp
+    ${UMC_CODECS}/vc1_dec/src/umc_vc1_video_decoder_hw.cpp
+    )
+
+list(APPEND sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/vp8/src/mfx_vp8_dec_decode_common.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/vp8/src/mfx_vp8_dec_decode_hw.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/vp8/src/mfx_vp8_dec_decode_hw_vaapi.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/vp8/src/mfx_vp8_dec_decode_tables.cpp
+    )
+
+list(APPEND sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/vp9/src/mfx_vp9_dec_decode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/vp9/src/mfx_vp9_dec_decode_hw.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/vp9/src/mfx_vp9_dec_decode_utils.cpp
+    ${UMC_CODECS}/vp9_dec/src/umc_vp9_bitstream.cpp
+    ${UMC_CODECS}/vp9_dec/src/umc_vp9_utils.cpp
+    ${UMC_CODECS}/vp9_dec/src/umc_vp9_va_packer.cpp
+    )
+
+if( MFX_ENABLE_SW_FALLBACK )
+    list(APPEND sources
+        ${UMC_CODECS}/color_space_converter/src/umc_video_processing.cpp
+        ${UMC_CODECS}/color_space_converter/src/umc_color_space_conversion.cpp
+        ${UMC_CODECS}/color_space_converter/src/umc_deinterlacing.cpp
+        )
+endif()
+
+set( sources.plus "" )
+
+set( defs "${API_FLAGS} ${WARNING_FLAGS}" )
```

Comment:
I believe we get rid of API_FLAGS and WARNING_FLAGS here on github. Please, don't introduce again.

Reply:
Done.